jobfunnel/backend/scrapers/indeed.py

```diff
@@ -225,22 +226,60 @@ def set(self, parameter: JobField, job: Job, soup: BeautifulSoup) -> None:
 
     def _get_search_url(self, method: Optional[str] = 'get') -> str:
         """Get the indeed search url from SearchTerms
+
+        Returns URL string for US and CANADA:
+        f"https://www.indeed.{_url}/jobs?q={_jobs}&l={_city}%2C+{_province}"
+        "&radius={_radius}&limit={_limit}&filter={_filters}{_remoteness}"
+
+        Returns URL string for UK or searching in USA or Canada for city:remote:
+        f"https://www.indeed.{_url}/jobs?q={_jobs}&l={_city}"
+        "&radius={_radius}&limit={_limit}&filter={_filters}{_remoteness}"
         TODO: use Enum for method instead of str.
         """
+        url = "https://www.indeed."
+        jobs = "/jobs?q="
+        location = "&l="
+        location_province = "%2C+"
+        radius = "&radius="
+        limit = "&limit="
+        filters = "&filter="
+
+        _url = self.config.search_config.domain
+        _jobs = self.query
+        _location_city = self.config.search_config.city.replace(' ', '+',)
+        _location_province = self.config.search_config.province_or_state.upper()
+        _radius = str(self._quantize_radius(self.config.search_config.radius))
+        _limit = str(self.max_results_per_page)
+        _filter_similar = str(self.config.search_config.return_similar_results)
+        _filter_remoteness = REMOTENESS_TO_QUERY[self.config.search_config.remoteness]
+
         if method == 'get':
-            return (
-                "https://www.indeed.{}/jobs?q={}&l={}%2C+{}&radius={}&"
-                "limit={}&filter={}{}".format(
-                    self.config.search_config.domain,
-                    self.query,
-                    self.config.search_config.city.replace(' ', '+',),
-                    self.config.search_config.province_or_state.upper(),
-                    self._quantize_radius(self.config.search_config.radius),
-                    self.max_results_per_page,
-                    int(self.config.search_config.return_similar_results),
-                    REMOTENESS_TO_QUERY[self.config.search_config.remoteness],
-                )
-            )
+            if _url == "co.uk":
```

Comment:
I think we should be using locale to determine this instead of a string comparison, ie if self.locale is ENGLISH_UK

Reply:
Right, I see how it would be much better to keep the organization. I reverted back to what you had. 

I now have a _get_search_url function in the IndeedScraperUKEng subclass that will overwrite BaseIndeedScraper. 

Same with having a _get_search_url in MonsterScraperUKEng overwritting BaseMonsterScraper.